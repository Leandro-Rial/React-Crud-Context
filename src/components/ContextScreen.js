import React from 'react'
import Heading from './Heading'
import UserList from './UserList'

const ContextScreen = () => {
    return (
        <>
            <Heading />
            <UserList />
        </>
    )
}

export default ContextScreen
