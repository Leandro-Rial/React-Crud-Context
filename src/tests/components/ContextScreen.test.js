import { shallow } from "enzyme";
import ContextScreen from "../../components/ContextScreen";

describe('Pruebas en ContextScreen', () => {
    test('debe de mostrarse correctamente', () => {
        const wrapper = shallow(
            <ContextScreen />
        )

        expect(wrapper).toMatchSnapshot()
    });
});