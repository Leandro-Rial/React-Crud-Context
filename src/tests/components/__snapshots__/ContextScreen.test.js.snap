// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en ContextScreen debe de mostrarse correctamente 1`] = `
<Fragment>
  <Heading />
  <UserList />
</Fragment>
`;
